ONSdigital/babbage
# Feature/new feedback form

### What

Added new feedback form. User feedback is now written (encrypted) to a configured location on the file system and an slack notification is sent to the appropriate channel. Notifications have the option of being throttled - set via babbage config.
### How to review

NOTE: This change will require the following config updates:
- Configure the location to write the feedback. For dev set _run-publishing.sh_ => `FEEDBACK_FOLDER` 
- Configure the public key to use to encrypt the feedback json files.  For dev update run-publishing.sh => `PUBLIC_KEY`

You will need also to run the zebedee `feature/feedback` (follow pull-req instructions) branch for this to work.

If you have done all this you should be able to submit feedback from any page (other than the home page) or from `/feedback`. If working correctly the user-feedback-test slack channel should receive a notification and an encrypted file and corresponding key file should be created in the folder you specified in  run-publishing.sh - both should be encrypted.
### Who can review

@CarlHembrough @ian-kent @LloydGriffiths
